src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/UnzipDirectoryChildren.cs

```diff
@@ -29,6 +29,8 @@ public UnzipDirectoryChildren ()
 		[Required]
 		public  ITaskItem       DestinationFolder   { get; set; }
 
+		public bool NoSubdirectory { get; set; }
```

Comment:
This change follows the "keep changes minimum" principle. If you make your suggested change, that affects other build targets. Such a semantic change should be done in separate PR.